src/braket/_sdk/__init__.py

```diff
@@ -11,4 +11,4 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-from ._version import __version__  # noqa: F401
+from ._version import __version__  # noqa: TID252, F401
```

Comment:
Is there a reason we prefer this over `braket._sdk._version`?

Reply:
I am not aware of the original decision. Any changes to this file outside of the addition of the noqa should be looked at in its PR